sdk-core/src/main/java/org/fisco/bcos/sdk/config/model/ConfigProperty.java

```diff
@@ -85,11 +91,35 @@ public static String getValue(Map<String, Object> config, String key, String def
         return (String) config.get(key);
     }
 
+    public static InputStream getConfigInputStream(String configFilePath) throws ConfigException {
+        if (configFilePath == null) {
+            return null;
+        }
+        InputStream inputStream = null;
+        try {
+            // try to load from the configFile
+            inputStream = new FileInputStream(configFilePath);
```

Comment:
try(inputStream = new FileInputStream(configFilePath)){}
for auto close

Reply:
done.